src/screens/Customers/CustomerProfile.tsx

```diff
@@ -126,35 +157,291 @@ function CustomerProfile(props: CustomerProps) {
+                <Typography variant="h3" className={classes.gray}>
+                  {meterInfo[0].number} {meterInfo[0].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Top Right */ }
+            <div style={{ float: 'left' }} className={classes.meterInfoContainer}>
+              <div style={{ marginLeft: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[2].label}</Typography>
+                <Typography variant="h3" className={classes.gray}>
+                  {meterInfo[2].number} {meterInfo[2].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Bottom Right */ }
+            <div style={{ float: 'right' }} className={classes.meterInfoContainer}>
+              <div style={{ marginLeft: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[3].label}</Typography>
+                <Typography variant="h3" className={classes.gray}>
+                  {meterInfo[3].number} {meterInfo[3].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Bottom Left */ }
+            <div style={{ float: 'right' }} className={classes.meterInfoContainer}>
+              <div style={{ marginRight: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[1].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[1].number} {meterInfo[1].unit}
+                </Typography>
+              </div>
+            </div>
+          </div>
+        </div>
+      );
+    }
+  }
+
   return (
     <BaseScreen leftIcon="backNav" title={customer.name} rightIcon="edit" match={match}>
       <BaseScrollView>
         <div className={classes.content}>
-          <Typography variant="h1">{currentSite.name}</Typography>
+          <Typography variant="h2">{currentSite.name}</Typography>
           <Typography variant="body1" color="textSecondary">
             {customer.meterNumber}
           </Typography>
-          <OutlinedCardList info={tariffInfo} primary={false} columns/>
-          <div className={classes.section}>
-            <div className={classes.headerWrapper}>
-              <Typography variant="h2">Payment</Typography>
-              {getRecordsButton()}
-            </div>
-            <OutlinedCardList info={balanceInfo} primary={true} rightIcon={getPaymentButton()} />
+          <div className={classes.infoContainer}>
+            {getTariffInfo()}
           </div>
-          <div className={classes.section}>
+          <div className={classes.headerWrapper}>
+            <Typography variant="h2">Payment</Typography>
+            <Link className={classes.buttonSecondary}
+              to={{
+              pathname: `${match.url}/records`,
+              state: { invoices: customer.meterReadings, payments: customer.payments, defaultTab: '1' },
```

Comment:
FYI the `customerData` PR got merged in, so `customer.meterReadings` isn't the way to access the meter readings / payments anymore. There PR shows the new way to grab this info, but lmk if you have trouble because the conflict is kinda my fault